gradle.properties

```diff
@@ -1,3 +1,3 @@
 group = org.threadly
-version = 2.0.0-SNAPSHOT
+version = 2.0.1-SNAPSHOT
```

Comment:
So the API changed (just additions), but I guess your right since there is no 2.0.0 release yet.